sdk/core/core-https/src/nodeHttpsClient.ts

```diff
@@ -4,7 +4,6 @@
 import * as https from "https";
 import * as zlib from "zlib";
 import { Transform } from "stream";
-import FormData from "form-data";
```

Comment:
@chradek I pulled the FormData stuff out into a policy per your advice

Reply:
Thank you thank you thank you!